riddhi_work/list_push_append.py

```diff
@@ -0,0 +1,26 @@
+# What is difference between push and append in list?
+
+
+# Program to add data in a list through append
+def list():
+    l = [1, 2, 3, 4, 5]
+    l.append(6)
+    return l
+
+
+print("Through append: " + str(list()))       #Output: [1, 2, 3, 4, 5, 6]
```

Comment:
why have you converted this in string
you can use `"Through append: {0}".format(l) ` read more over here https://pyformat.info/

PS it won't print the O/P you showed in comment